core/src/main/java/se2/ticktackbumm/core/models/cards/Card.java

```diff
@@ -1,104 +1,175 @@
-        backsideImage = new Image(backsideTexture);
+        font = new BitmapFont();
+        font.setColor(Color.CORAL);
+        font.getData().setScale(7);
+        font.getRegion().getTexture().setFilter(Texture.TextureFilter.Linear, Texture.TextureFilter.Linear);
+
+
+        assetManager.load("card/backside.png",Texture.class);
+        assetManager.finishLoading();
+
+        backsideTexture = assetManager.get("card/backside.png",Texture.class);
+
 
-        // TODO: add switch over game modes?
-        gameData.setCurrentGameModeText(getRandomSyllable());
-        // TODO: testing only
-        gameData.setCurrentGameModeText("UNG"); // set syllable always to UNG for testing
+        backSprite = new Sprite(backsideTexture);
 
-        // TODO: load and get with AssetManager
-        frontsideTexture = assetManager.get("card/frontside.png", Texture.class);
-        frontsideImage = new Image(frontsideTexture);
+        assetManager.load("card/frontside.png",Texture.class);
+        assetManager.finishLoading();
+
+        frontsideTexture = assetManager.get("card/frontside.png",Texture.class);
+
+        randomWord = getWordDependOnMode();
+
+        fontSprite = new Sprite(frontsideTexture);
+        fontSprite.setBounds(Gdx.graphics.getWidth() / 2.0f - 200, Gdx.graphics.getHeight() / 2.0f, 500, 300);
 
-        // TODO: use skin instead of LabelStyle
-        cardWordLabel = new Label(gameData.getCurrentGameModeText(), new Label.LabelStyle(new BitmapFont(), Color.WHITE));
 
-        setupBackSide();
     }
 
-    public void draw() {
-        if (Gdx.input.isTouched()) {
-            setupFrontSide();
+    public void drawCard(SpriteBatch spriteBatch) {
+        if (Gdx.input.isTouched()){
+            if (isRevealed){
+                isRevealed = false;
+            }
+            else{
+                isRevealed = true;
+            }
+        }
+        if (isRevealed){
+            drawFrontSide(spriteBatch);
+
         }
+        else{
+           drawBackSide(spriteBatch);
+        }
+    }
 
-        stage.draw();
+    public void drawBackSide(SpriteBatch spriteBatch){
+        backSprite.setBounds(Gdx.graphics.getWidth() / 2.0f - 200, Gdx.graphics.getHeight() / 2.0f, 500, 300);
+        spriteBatch.begin();
+        backSprite.draw(spriteBatch);
+        spriteBatch.end();
     }
 
-    // TODO: refactor draw methods/render methods
-    public void setupBackSide() {
-        setActorSettings(backsideImage, Gdx.graphics.getWidth() / 2.0f - 200, Gdx.graphics.getHeight() / 2.0f, 400, 200);
-        stage.addActor(backsideImage);
+    public void drawFrontSide(SpriteBatch spriteBatch){
+        spriteBatch.begin();
+        fontSprite.draw(spriteBatch);
+        font.draw(spriteBatch,randomWord,Gdx.graphics.getWidth() / 2.0f - 28, Gdx.graphics.getHeight() / 2.0f+203);
+        spriteBatch.end();
+
+        this.game.getNetworkClient().getClientMessageSender().sendCardOpened();
```

Comment:
Vorsicht, diese Methode wird in einer `render()`-Methode aufgerufen. Möglicherweise wird diese Nachricht dann 60 mal in der Sekunde an den Server gesendet!